#white_box   : {box: [0.1, 0.1, 0.1], color: [255,255,255,1.]}
#orange_box  : {box: [0.1, 0.1, 0.1], color: [253.,173.,5.,1.]}
#blue_box    : {box: [0.1, 0.1, 0.1], color: [38.,93.,107.,1.]}

white_box   :
 box: [0.05, 0.05, 0.05]
 color: [1,1,1,1.]
 offset: [0,0,-0.025]
 grasp_poses:
 - {tool: "tip", position: [0, 0, 0.005], quaternion: [1,0,0,0]}


orange_box:
  box: [0.05, 0.05, 0.05]
  color: [1,0.8,0,1.]
  offset: [0,0,-0.025]
  grasp_poses:
  - {tool: "tip", position: [0, 0, 0.005], quaternion: [1,0,0,0]}

blue_box:
  box: [0.05, 0.05, 0.05]
  color: [0.2,0.3,0.7,1.0]
  offset: [0,0,-0.025]
  grasp_poses:
  - {tool: "tip", position: [0, 0, 0.005], quaternion: [1,0,0,0]}








